ci/ci.sh

```diff
@@ -8,7 +8,7 @@ cd ci
 
 ./setup_conan.sh
 
-export VITASDK=$(pipenv run conan info vitasdk-toolchain/06@blastrock/pkgj --paths -pr vita | grep -Po '(?<=package_folder: ).*$')
+export VITASDK=$(pipenv run conan info vitasdk-toolchain/07@blastrock/pkgj --paths -pr vita | grep -Po '(?<=package_folder: ).*$')
```

Comment:
Sorry for that, it's already fixed on my branch

Reply:
Cool!